api/request.go

```diff
@@ -42,3 +44,14 @@ func GetAPIRequest(r *http.Request) *Request {
 	}
 	return nil
 }
+
+// TextResponse writes a text response.
+func TextResponse(w http.ResponseWriter, r *http.Request, text string) {
+	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
+	w.Header().Set("X-Content-Type-Options", "nosniff")
+	w.WriteHeader(http.StatusOK)
+	_, err := fmt.Fprintln(w, text)
+	if err != nil {
```

Comment:
`Fprintln` adds a newline at the end of the stream. Is this on purpose?

Reply:
I just followed how `http.Error` does it, so yes!